LayoutTests/platform/glib/TestExpectations

```diff
@@ -2868,10 +2869,6 @@ fast/mediastream/captureInGPUProcess.html [ Skip ]
 fast/mediastream/video-rotation-gpu-process-crash.html [ Skip ]
 http/wpt/mediarecorder/MediaRecorder-AV-audio-video-dataavailable-gpuprocess.html [ Skip ]
 webgl/webgl-fail-after-context-creation-no-crash.html [ Skip ]
-fast/mediastream/mediastreamtrack-clone-muted.html [ Skip ]
```

Comment:
So this test for instance, where is it explicitely requesting GPUP-specific features?

Reply:
I don't know where it's enabled for that particular test.

Reply:
According to @youennf capture devices access has to be done from the GPUP if it's enabled at build time, iiuc...

Reply:
The trigger here comes from `testRunner.setMockCaptureDevicesInterrupted()`, `triggerMockCaptureConfigurationChange()` is another one, maybe there are more, not sure... so currently any test using those functions is going to crash... 

If only these 2 functions are problematic then we're not in the hundreds range, I see only 8 tests using those.